src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/ManageUsersApiExceptionHandler.kt

```diff
@@ -128,52 +124,6 @@ class HmppsManageUsersApiExceptionHandler {
       .body(ErrorResponse(status = BAD_REQUEST.value(), developerMessage = e.message))
   }
 
-  @ExceptionHandler(RoleNotFoundException::class)
```

Comment:
just make sure these are add as a part of auth refactor and none of manager user UI relies on these error codes

Reply:
Done